torch/csrc/jit/codegen/cuda/fusion_segmenter.cpp

```diff
@@ -1937,22 +1937,33 @@ void TranslateApplicableWelford::translateSingleWelford(WelfordOp* welford) {
   auto out_N = welford->outN()->as<TensorView>();
 
   fusion->removeExpr(welford);
+  // Not safe to use welford anymore
+  welford = nullptr;
```

Comment:
It'd be helpful if `removeExpr` could automate this.

Reply:
Yeah, but seems non-trivial unless we make all pointers be some sort of smart pointers.